PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -305,15 +321,24 @@ DECLARE_SOA_DYNAMIC_COLUMN(P, p, //!
 } // namespace reducedpair
 
 DECLARE_SOA_TABLE(Dileptons, "AOD", "RTDILEPTON", //!
-                  reducedpair::ReducedEventId, reducedpair::Mass,
+                  reducedpair::ReducedEventId,
+                  reducedpair::Mass,
                   reducedpair::Pt, reducedpair::Eta, reducedpair::Phi, reducedpair::Sign,
                   reducedpair::FilterMap,
                   reducedpair::Px<reducedpair::Pt, reducedpair::Phi>,
                   reducedpair::Py<reducedpair::Pt, reducedpair::Phi>,
                   reducedpair::Pz<reducedpair::Pt, reducedpair::Eta>,
                   reducedpair::P<reducedpair::Pt, reducedpair::Eta>);
 
+DECLARE_SOA_TABLE(DileptonsExtra, "AOD", "RTDILEPTONEXTRA", //!
+                  dilepton_track_index::Index0Id, dilepton_track_index::Index1Id,
+                  reducedpair::Rap,
```

Comment:
Please remove the rapidity column from this table. Here we should keep just "extra" information, which in this case is secondary vertexing info.